client/product_details/DefaultView.jsx

```diff
@@ -1,31 +1,26 @@
 import React from 'react';
-import ThumbnailList from './ThumbnailList.jsx';
 
-class DefaultView extends React.Component {
-  constructor(props) {
-    super(props);
-    
-    this.state = {
-      currPhotoIndex: 0,
-    };
-  }
+const DefaultView = (props) => {
 
-  render() {
-    if (this.props.selectedStyle !== undefined) {
-      var photoGallery = this.props.selectedStyle.photos;
-      return (
-        <div id="defaultView">
-          <a target="_blank" href={photoGallery[this.state.currPhotoIndex].url}>
-            <img src={photoGallery[this.state.currPhotoIndex].url} alt={this.props.selectedStyle.name} className="defaultView" id={this.props.selectedStyle.style_id}></img>
-          </a>
-        </div>
-      );
-    } else {
-      return (
-        <div id="defaultView">DEFAULT VIEW</div>
-      );
-    }
+  if (props.selectedStyle !== undefined) {
+    var photoGallery = props.selectedStyle.photos;
+    return (
+      <div id="defaultView">
+        <a target="_blank">
+          <img 
+            src={photoGallery[props.currPhotoIndex].url} 
+            alt={props.selectedStyle.name} 
+            className="defaultView" 
+            id={props.selectedStyle.style_id}>
+          </img>
+        </a>
+      </div>
+    );
+  } else {
```

Comment:
I think the "else" part of the statement may be unnecessary since the "if" will return something and not run this code

Reply:
Good catch! I'll fix this on the next PR